### PR TITLE
Chia devtools lint cleanup 

### PR DESCRIPTION
**Pull request by jack60612**
- [x] Switch to Chia 1.5.0
- [x] Fix all tests that broke because of cat2
- [x] Fix all errors from black, flake8, mypy and run isort 
- [x] Add **Working** automated CI runners

### CODE REVIEW

**Comment by jack60612** [comment]
ignore the force pushes, i did a bit of a git oops

**Comment by mariano54** [comment]
LGTM, but probably should get quex to approve since he owns this code

File: cdv/cmds/rpc.py [hunk 1]
```diff
@@ -273,8 +271,11 @@ async def do_command():
         try:
             node_client: FullNodeRpcClient = await get_client()
             if transaction_id:
-                items = {}
-                items[transaction_id] = await node_client.get_mempool_item_by_tx_id(hexstr_to_bytes(transaction_id))
+                items = {
+                    transaction_id: await node_client.get_mempool_item_by_tx_id(
+                        bytes32.from_bytes(hexstr_to_bytes(transaction_id))
```
Comment (Quexington) [inline cdv/cmds/rpc.py:276]: Consistent with the other changes, this should be bytes32.from_hexstr yeah?

**Review by Quexington — APPROVED** [review]
Seems fine probably

File: cdv/cmds/rpc.py (hunk 1 above)
Comment (jack60612) [reply cdv/cmds/rpc.py:276]: Passes lint so I'm not sure, I'll leave it as is.

Comment (jack60612) [reply cdv/cmds/rpc.py:276]: Oh wait yep I see what you mean

**Pull request merged**